templates/default/flake.nix

```diff
@@ -14,7 +14,7 @@
     flake-utils.lib.eachDefaultSystem (system:
       let
         pkgs = nixpkgs.legacyPackages.${system};
-        terraform = nixpkgs-terraform.packages.${system}."1.7.2";
+        terraform = nixpkgs-terraform.packages.${system}."1.7.4";
```

Comment:
@oscar-izval It appears that there is some room for improvement here. I believe that eventually the `update-version.py` script should update all templates. What do you think?